### PR TITLE
add jellyfin-web as submodule

### PR DESCRIPTION
**Pull request by dkanada**
With the UI in a separate repo the client apps can pull it as well. This means we can update the old versions of the UI in android, samsungtv, and a few others to use the same codebase across all projects.

### CODE REVIEW

**Comment by EraYaN** [comment]
We might want to rename the submodule to `jellyfin-web` cause it might have a different path anyway internally. `src` for example and the thing we want to ship will be in `dist` when we can a proper bundler (webpack4 etc.) script going.

**Comment by joshuaboniface** [comment]
I agree with @EraYaN if we're going to do it now, let's do it right.

**Comment by anthonylavado** [comment]
Just a note, I'll be pulling some issues out of the 10.0.0 release as blocked by this.

We can address them with maintenance releases after the big one.

**Comment by cvium** [comment]
@anthonylavado I don't think that's a good idea. There are a bunch of UI fixes that are "blocked" (I guess?) by this change. I don't think it makes sense to releae 10.0.0 with known UI bugs. Why not wait a little instead?

**Comment by anthonylavado** [comment]
@jellyfin/core We need to weigh in on this

**Comment by anthonylavado** [comment]
@cvium: What's the most broken that needs to be fixed up?

**Comment by dkanada** [comment]
We could technically hold this PR and still merge the others, it just seems slightly inefficient to keep pushing UI changes and other things to this repo if we plan on moving the it to another repository. Another thing to note is that we recently realized a lot of the previous use cases for the separate repo will be null and void. The cordova client in its current form can't support iOS, and several other client apps that may have been using the web interface might not need to bundle it with the releases.

**Comment by EraYaN** [comment]
https://github.com/jellyfin/jellyfin-web/tree/dev This one is now ready.

**Comment by EraYaN** [comment]
@anthonylavado @joshuaboniface Just updated the jellyfin-web repo. This might need merging.

File: MediaBrowser.WebDashboard/Api/DashboardService.cs [hunk 1]
```diff
@@ -150,7 +150,7 @@ public string DashboardUIPath
                     return _serverConfigurationManager.Configuration.DashboardSourcePath;
                 }
 
-                return Path.Combine(_serverConfigurationManager.ApplicationPaths.ApplicationResourcesPath, "dashboard-ui");
+                return Path.Combine(_serverConfigurationManager.ApplicationPaths.ApplicationResourcesPath, "jellyfin-web/src");
```
Comment (JustAMan) [inline MediaBrowser.WebDashboard/Api/DashboardService.cs]: Are you sure this is correct? I don't see `src` subfolder in the submodule repo at all, but I see `dashboard-ui` in there.

File: MediaBrowser.WebDashboard/MediaBrowser.WebDashboard.csproj [hunk 2]
```diff
@@ -10,7 +10,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="dashboard-ui\**\*.*">
+    <None Include="jellyfin-web\src\**\*.*">
```
Comment (JustAMan) [inline MediaBrowser.WebDashboard/MediaBrowser.WebDashboard.csproj]: same

File: MediaBrowser.WebDashboard/Api/DashboardService.cs (hunk 1 above)
Comment (Bond-009) [inline MediaBrowser.WebDashboard/Api/DashboardService.cs]:
```suggestion
                return Path.Combine(_serverConfigurationManager.ApplicationPaths.ApplicationResourcesPath, "jellyfin-web", "src");
```
Stay consisted.

**Pull request merged**